usecases/traverser/explorer.go

```diff
@@ -308,6 +313,10 @@ func (e *Explorer) getClassList(ctx context.Context,
 	} else {
 		res, err = e.search.ClassSearch(ctx, params)
 		if err != nil {
+			var e inverted.MissingIndexError
```

Comment:
Same remark as before

Reply:
This change was introduced to address the issue https://github.com/weaviate/weaviate/issues/2974.
Error was simplified and made human-friendly only for one particular case: index is not configured for property used in gql query, which may be quite common case for beginner users. All other, less probable, errors will have still full message returned.